sdk/react/panels/videoView.js

```diff
@@ -121,7 +121,13 @@ class VideoView extends React.Component {
   };
 
   onSeekPressed = (isForward) => {
-    const seekValue = (isForward) ? this.props.config.skipControls.skipForwardTime : this.props.config.skipControls.skipBackwardTime * (-1);
+    let configSeekValue = (isForward) ? this.props.config.skipControls.skipForwardTime : this.props.config.skipControls.skipBackwardTime;
+    if (configSeekValue > 99) {
```

Comment:
Would be great to extract `1` and `99` since we are using it in 2 different files